packages/gatsby-plugin-gatsby-cloud/src/components/Indicator.js

```diff
@@ -1,19 +1,87 @@
 import React, { useState, useEffect, useCallback, useRef } from "react"
-
 import getBuildInfo from "../utils/getBuildInfo"
+import IndicatorButton from "./IndicatorButton"
+import getGatsbyIndicatorButtonProps from "./getGatsbyIndicatorButtonProps"
+import getLinkIndicatorButtonProps from "./getLinkIndicatorButtonProps"
+import getInfoIndicatorButtonProps from "./getInfoIndicatorButtonProps"
+import { gatsbyIcon, infoIcon, linkIcon, successIcon } from "./icons"
 import Style from "./Style"
 
-import GatsbyIndicatorButton from "./GatsbyIndicatorButton"
-import LinkIndicatorButton from "./LinkIndicatorButton"
-import InfoIndicatorButton from "./InfoIndicatorButton"
-
 const POLLING_INTERVAL = process.env.GATSBY_PREVIEW_POLL_INTERVAL || 3000
 
+export function PreviewIndicator({
```

Comment:
Is this a good name? Should we do something like `UNSAFE_PreviewIndicator` until we have this API in a place where we feel it is more stable?

Reply:
we use the unstable_ prefix but this will only be used internally. So, it's not necessary